Kickstarter-iOS/ViewModels/AppDelegateViewModelTests.swift

```diff
@@ -1365,6 +1365,39 @@ final class AppDelegateViewModelTests: TestCase {
     self.goToMobileSafari.assertValues([unrecognizedUrl], "Go to mobile safari for the unrecognized url.")
   }
 
+  func testEmailDeepLinking_UnrecognizedUrl_ProjectPreview() {
```

Comment:
This test is essentially the same as the one above it but it explicitly uses a known project preview link (containing `token` query param) to test that it will be ignored.